hack/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -0,0 +1,1142 @@
+---
```

Comment:
One last question: why we have these crds generated now since I last time reviewed, what has changed?

Reply:
Sorry, due to switching between multiple commits/PRs I accidentally added what is gitignored in https://github.com/rancher-sandbox/rancher-turtles/pull/84 and allows us to support structured types and run tests. Fixrd